tests/tests_bond_deprecated_initscripts.yml

```diff
@@ -2,13 +2,15 @@
 # This file was generated by ensure_provider_tests.py
 ---
 - hosts: all
-  name: Run playbook 'playbooks/tests_bond_deprecated.yml' with initscripts
+  name: Run playbook 'playbooks/tests_bond_deprecated.yml' with initscripts as provider
   tasks:
-    - include_tasks: tasks/el_repo_setup.yml
```

Comment:
why remove epel setup?

Reply:
I did not explicitly remove the epel setup, but `include_tasks: tasks/el_repo_setup.yml` is not appear in   https://github.com/linux-system-roles/network/blob/main/tests/ensure_provider_tests.py#L116

Reply:
Let us add `- include_tasks: tasks/el_repo_setup.yml` in ensure_provider_tests.py

Reply:
Updated